.github/workflows/linters.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          export PYTHONPATH="${PYTHONPATH}:/erigon_server"
```

Comment:
As you should see by looking at the CI checks, this didn't work. There is no `/erigon_server` directory. Also, this approach is mistaken because every step in a workflow is run in a separate shell. Please read [this](https://github.community/t/set-pythonpath-in-github-actions-running-on-ubuntu/17939) for some tips.